CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Added new MessageInputView structure
 - Added support for marking all channels as read for the current user.
     - Can be accessed via `ChatDomain`'s use cases (`chatDomain.useCases.markAllRead()...`).
 - Fix bug when local channels could be sorted not properly
+- Typing events can be all tracked from `ChatDomain`
```

Comment:
Looking at this as a user, I don't know _how_ to track typing events. Would be useful to indicate the specific class/method to be used.

Reply:
Done